test/journeys/lib/test-helpers/space-widget/meet.js

```diff
@@ -57,3 +63,101 @@ export function hangup(aBrowser) {
   aBrowser.waitForVisible(elements.callControls);
   aBrowser.element(elements.meetWidget).element(elements.hangupButton).click();
 }
+
+/**
+ * Test to hangup call before reciever answers
+ * @param {Object} browserLocal
+ * @param {Object} browserRemote
+ * @returns {void}
+ */
+export function hangupBeforeTest(browserLocal, browserRemote) {
```

Comment:
Yeah, you're right. Want me to change `hangupDuringTest` and `declineIncomingTest` to `hangupDuringCallTest` and `declineIncomingTest` or are those okay?